templates/creator/_edit.twig

```diff
@@ -382,7 +382,7 @@
                                         heading: "Contact Type" |t,
                                         type: "select" |t,
                                         options: {
-                                            "customer support": "Customer Support",
```

Comment:
Changed to follow Google's standard: https://developers.google.com/search/docs/data-types/corporate-contacts